Application/Features/WorkplaceFeature/Commands/CreateWorkplaceCommand.cs

```diff
@@ -55,7 +55,7 @@ public async Task<CreateWorkplaceCommandResponse> Handle(CreateWorkplaceCommandR
             }
             
 
-            _context.Workplaces.Add(workplace);
+            await _context.Workplaces.AddAsync(workplace, cancellationToken);
```

Comment:
wrong, context AddAsync should not be used, please look at documentation

Reply:
changed to "_context.Workplaces.Add(workplace);"